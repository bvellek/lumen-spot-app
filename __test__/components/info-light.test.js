import React from 'react';
import renderer from 'react-test-renderer';

import InfoLight from '../../public/js/components/info-light';

describe('<InfoLight />', () => {
  it('it renders correctly', () => {
    const component = renderer.create(
      <InfoLight />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
