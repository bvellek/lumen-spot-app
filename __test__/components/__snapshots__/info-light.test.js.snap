// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoLight /> it renders correctly 1`] = `
<section
  className="info-section"
>
  <div
    className="light-section"
  >
    <h2>
      The Sun
    </h2>
    <p>
      The Sun is the most critical light source we have. Looking to capture breathtaking portraits at that golden hour or take advantage of the blue hour for cold landscapes? Read on to find out more about what the sun times we provide mean to you.
    </p>
    <p>
      <span>
        Golden Hour
      </span>
      : the time when the sky is filled with golden red-yellow tones. Diffused light with low contrast has a warming effect on your photos. This magical condition occurs briefly in the morning from the end of civil twilight through the beginning of the sunrise. In the evening you can catch the golden hour toward the end of sunset and into the beginning of civil twilight.
    </p>
    <p>
      <span>
        Blue Hour
      </span>
      : the time just before golden hour in the morning (beginning of civil twilight) and just after, in the evening (end of civil twilight). The sky’s color will be cool and saturated, filled with deep blue. This can be a great time for urban photography, moon shots, and cool landscapes.
    </p>
    <p>
      <span>
        Twilights
      </span>
      : In the morning the sun rises through astronomical twilight, nautical twilight, civil twilight and the full sunrise. In the evening and into the night the sun sets to civil twilight, nautical twilight, astronomical twilight and into nighttime.
    </p>
    <ul>
      <li>
        <span>
          Civil Twilight
        </span>
        : where the golden and blue hours can be found.
      </li>
      <li>
        <span>
          Nautical Twilight
        </span>
        : provides diffuse, blue light which can be great for capturing silhouettes or longer exposure shots.
      </li>
      <li>
        <span>
          Astronomical Twilight
        </span>
        : darker sky and more visible stars. This is the time for those beautiful shots of the milky way.
      </li>
    </ul>
  </div>
</section>
`;
